### DISCUSSION TITLE
Dual language site

### DISCUSSION THREAD

**Discussion by craigvear** [jekyll/jekyll#9291] [Q&A] [answered]
Hi all,

I have a research project that I am hosting using Jekyll and GitHub. https://digiscore.github.io

 I’m wanting to have a parallel site embedding in it for Chinese language : https://digiscore.github.io/ch

is it possible to have a duplicate _config and home layout where the menu options are also in Chinese?

I could then direct them to the translated pages. The blogs would remain the same.

bw
Craig

**Comment by 0962692862** [comment]
. 👎 👎

ตาตี่

ในวันที่ จ. 6 ก.พ. 2023 00:43 Craig Vear ***@***.***>
เขียนว่า:

**Comment by craigvear** [comment, marked as answer]
Went for a drop down menu option instead. Seems to do the job.
